opentracing-api/src/main/java/io/opentracing/propagation/Binary.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2016-2018 The OpenTracing Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.opentracing.propagation;
+
+import java.nio.ByteBuffer;
+
+/**
+ * Binary is an interface defining the required operations for a binary carrier for
+ * Tracer.inject() and Tracer.extract(). Binary can be defined either as inbound (extraction)
+ * or outbound (injection).
+ *
+ * When Binary is defined as inbound, extractionBuffer() will be called to retrieve the ByteBuffer
+ * containing the data used for SpanContext extraction.
+ *
+ * When Binary is defined as outbound, setInjectBufferLength() will be called in order to hint
```

Comment:
There is no setInjectBufferLenght() method defined in the interface, so it seems to me the documentation does not match the implementation here.

Reply:
Oh, definitely. Will update, thanks for the observation.